sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/aml_compute.py

```diff
@@ -130,6 +130,7 @@ def __init__(
             name=name,
             description=description,
             location=kwargs.pop("location", None),
+            tags=kwargs.pop("tags", None),
```

Comment:
Add doctoring for tags?

Reply:
Added to the compute subtype constructors.